src/cmd-buildextend-metal

```diff
@@ -271,7 +259,8 @@ runvm "${target_drive[@]}" -- \
             --firstboot-kargs "\"${firstboot_kargs}\"" \
             --kargs "\"${kargs}\"" \
             --osname "${name}" \
-            --ostree-ref "${ref_arg}" \
+            --ostree-commit "${commit}" \
+            --ostree-ref "${ref:-NONE}" \
```

Comment:
This ugly hack would go away with https://github.com/coreos/coreos-assembler/pull/1787